Framework/Crystal/src/SCDPanelErrors.cpp

```diff
@@ -209,13 +221,11 @@ void SCDPanelErrors::eval(double xshift, double yshift, double zshift,
       out[i * 3 + 1] = Q3[1] - Q2[1];
       out[i * 3 + 2] = Q3[2] - Q2[2];
     } catch (std::runtime_error &) {
-      out[i * 3] = std::numeric_limits<double>::infinity();
-      out[i * 3 + 1] = std::numeric_limits<double>::infinity();
-      out[i * 3 + 2] = std::numeric_limits<double>::infinity();
+      out[i * 3] = 0.15;     // std::numeric_limits<double>::infinity();
```

Comment:
Should remove the commented out infinities from lines 224-226.
```suggestion
      out[i * 3] = 0.15;
```